src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,16 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment: { name, email, body } }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{name}</strong>
+
+      {' by '}
```

Comment:
it's bad practice to add spaces this way, better to use margins and padding for that